homeassistant/components/light/zwave.py

```diff
@@ -244,9 +235,9 @@ def _value_added(self, value):
         # Check for the missing color values
         self._get_color_values()
 
-    def update_properties(self):
+    def update(self):
```

Comment:
I think that we should try to keep components and platform similar in code structure and reuse patterns whenever we can, to make it easier to know what's going on. Here, using `update` for updates should be the standard in my book. But if @balloob really think the name change was a bad idea in this case, I'll rest my case.